docs/projects/run-build-deploy.md

```diff
@@ -84,10 +89,31 @@ run2 = project.run_function("train", inputs={"dataset": run1.outputs["data"]})
 run2.artifact('confusion-matrix').show()
 ```
 
-```{admonition} Run/simulate functions locally: 
-Functions can also run and be debugged locally by using the `local` runtime or by setting the `local=True` 
-parameter in the {py:meth}`~mlrun.runtimes.BaseRuntime.run` method (for batch functions).
+Example of `new_function` with `new_task`:
```

Comment:
Generally speaking, we want to shift from using the global `new_function`/`run_function` method to project methods. Since we're making changes here, this may be a good opportunity to do it.
The only function that we don't have a project equivalent for is the `new_task` function.

Reply:
- replaced 2 instances of  `...as project methods, or as global (mlrun.) methods` with `...as project methods`
- updated new_task example